.changeset/dirty-geese-wait.md

```
@@ -0,0 +1,12 @@
+---
+'@mastra/core': minor
+---
+
+Added AgentController live-session deletion with a process-local listener.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)